app/code/Magento/PageCache/Observer/FlushFormKey.php

```diff
@@ -22,13 +23,20 @@ class FlushFormKey implements ObserverInterface
     private $dataFormKey;
 
     /**
+     * @var Session
+     */
+    private $customerSession;
+
+    /**
      * @param CookieFormKey $cookieFormKey
      * @param DataFormKey $dataFormKey
+     * @param Session $customerSession
      */
-    public function __construct(CookieFormKey $cookieFormKey, DataFormKey $dataFormKey)
+    public function __construct(CookieFormKey $cookieFormKey, DataFormKey $dataFormKey, Session $customerSession)
```

Comment:
This is not backward compatible change. Could you add $customerSession as optional parameter with fallback to retrieving this object via ObjectManger?

Reply:
Thanks @ihor-sviziev I will work after mm17es in this fix.